docs/content/temporary-components.md

```diff
@@ -0,0 +1,43 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+
+
+## Temporary components
```

Comment:
Still think that some kind of motivation as to why we need temporary components is missing here. The doc just jumps straight to the implementation

Reply:
That's mostly because we are missing all the things that go with it and the three or so docs that also need to be included. This is factored out from other stuff.
The reason for it to be there is mostly the temporary components on the worker entity and also a prerequisite for the timer systems and various other things.